website/docs/ref/react.doc.js

```diff
@@ -197,6 +197,28 @@ function renderButton(container: HTMLElement, visited?: boolean) {
 
   Any function that returns a React element can be used as a component class in
   a JSX expression.
+
+*/
+const AsyncGreeter = loadAsync(Greeter, new Promise((resolve, reject) => {
```

Comment:
@nmn Can you create a separate issue for that if one does not already exist and link it here? We should discuss that on its own